pkg/identity/metricsd.go

```diff
@@ -0,0 +1,65 @@
+package identity
+
+import (
+	"strings"
+	"time"
+
+	internal "github.com/AthenZ/k8s-athenz-sia/pkg/metrics"
```

Comment:
`internal` package is a special keyword

Reply:
please add a comment for it

Reply:
Since this is a general knowledge of golangI'd rather not re-explain in Athenz source code.
https://github.com/golang/go/wiki/Modules#can-a-module-depend-on-an-internal-in-another